src/main/java/com/minecolonies/coremod/network/messages/BuildToolPlaceMessage.java

```diff
@@ -124,36 +125,39 @@ public void toBytes(@NotNull final ByteBuf buf)
     public void messageOnServerThread(final BuildToolPlaceMessage message, final EntityPlayerMP player)
     {
         final World world = player.world;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move the declaration of "world" closer to the code that uses it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1941)